pmp_iv/models/eda.py

```diff
@@ -0,0 +1,46 @@
+import csv
+from pmp_iv.models.correlacion_area import *
+class EDA():
+
+    def __init__(self):      
+        with open('data/incendiosForestales.csv', 'r', newline='') as csvfile:       
+            self.weather_data = csv.DictReader(csvfile)
+            self._datos_csv = self.get_data_values()
+            self._area = self.by_property('area')
+ 
+    def get_data_values(self):
+        filter_data = []
+        for row in self.weather_data:
+            my_dict = {}
+            for key, value in row.items():
+                if key != 'month' and key != 'day':
+                    my_dict[key] = float(value)
+                else:
+                    my_dict[key] = value
+            filter_data.append(my_dict)
+        return filter_data
+    
+    def by_date_property(self, mes, dia, propiedad):
+        data_by_property = []
+        for i in self.valores:
```

Comment:
Debes tratar de evitar los bucles for todo lo posible. Usa `map` o `filter` o construcciones similares.